storybookjs/storybook
# docs(examples): Add Auth0 Cosmos to list of examples

Issue:

## What I did

Added [Auth0 Cosmos](https://auth0-cosmos.now.sh/) to [/examples](https://storybook.js.org/examples/) page

## How to test

Is this testable with Jest or Chromatic screenshots?
Does this need a new example in the kitchen sink apps?
Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["documentation"]`
